public/reportissue.php

```diff
@@ -68,57 +68,65 @@ function displayCore() {
             <table>
                 <tbody>
                 <tr>
-                    <td>Game:</td>
+                    <td>Game</td>
                     <td style="width:80%">
                         <?php echo GetGameAndTooltipDiv($gameID, $gameTitle, $gameBadge, $consoleName, false) ?>
                     </td>
                 </tr>
                 <tr>
-                    <td>Achievement:</td>
+                    <td>Achievement</td>
                     <td>
                         <?php echo GetAchievementAndTooltipDiv(
-    $achievementID,
-    $achievementTitle,
-    $desc,
-    $achPoints,
-    $gameTitle,
-    $achBadgeName,
-    true
-) ?>
+                            $achievementID,
+                            $achievementTitle,
+                            $desc,
+                            $achPoints,
+                            $gameTitle,
+                            $achBadgeName,
+                            true)
+                        ?>
                     </td>
                 </tr>
                 <tr class="alt">
-                    <td><label for="issue">Issue:</label></td>
+                    <td><label for="issue">Issue</label></td>
                     <td>
                         <select name="p" id="issue" required>
                             <option value="" disabled selected hidden>Select your issue...</option>
                             <option value="1">Triggered at wrong time</option>
                             <option value="2">Doesn't Trigger</option>
                         </select>
+                        <a href="/views/issueDescriptionModal.html" rel="modal:open">?</a>
                     </td>
                 </tr>
                 <tr>
-                    <td><label for="emulator">Emulator:</label></td>
+                    <td><label for="emulator">Emulator</label></td>
                     <td>
-                        <select name="note[emulator]" id="emulator" onchange="displayCore()" required>
+                        <select name="note[emulator]" id="emulator" required data-bind="value: emulatorValue">
                             <option value="" disabled selected hidden>Select your emulator...</option>
                             <?php foreach ($emulators as $emulator): ?>
                                 <option><?= $emulator['handle'] ?></option>
                             <?php endforeach ?>
                         </select>
                     </td>
                 </tr>
-                <tr id="core-row" style="display: none">
+                <tr>
+                    <td><label for="version">Emulator Version</label></td>
+                    <td>
+                        <input type="text" name="note[emulatorVersion]" id="version" required />
+                        <a href="/views/versionDescriptionModal.html" rel="modal:open">Why?</a>
```

Comment:
I'd prefer inlining those modals instead of separate files. Not too sure about that though.

Reply:
Strong disagree there. It's way better to split up views to make them easier to digest for future devs and even for people like me, who like to compartmentalize everything into small files.